core/src/test/java/utils/AddressTest.java

```diff
@@ -0,0 +1,42 @@
+package utils;
+
+import org.junit.jupiter.api.Test;
+import org.nervos.ckb.address.Network;
+import org.nervos.ckb.type.Script;
+import org.nervos.ckb.utils.Numeric;
+import org.nervos.ckb.utils.address.Address;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AddressTest {
+  private Script script = new Script(
+      Numeric.hexStringToByteArray("0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"),
+      Numeric.hexStringToByteArray("0xb39bbc0b3673c7d36450bc14cfcdad2d559c6c64"),
+      Script.HashType.TYPE);
+
+  @Test
+  @SuppressWarnings("deprecation")
+  public void testEncode() {
```

Comment:
please add more test case, you may use the rfc demo code as fixture: https://github.com/rev-chaos/ckb-address-demo